README.md

```diff
@@ -1,5 +1,10 @@
 # IDSEQ-DAG
 
+idseq_dag is the pipeline execution engine for idseq (see idseq.net). It's a dag based pipelining system where the pipeline could be composed with indivisual python classes through json.
+
+The pipeline would be run on where you execute them with local machine resources. More details could be found below.
```

Comment:
First sentence is a bit confusing... are you trying to say "The pipeline runs locally."?

Reply:
yes.